tools/fonts/inspector/otf.html

```diff
@@ -123,35 +123,47 @@
       // We can derive the OTF file path automatically from the font name.
       const musicFonts = [
         'Bravura',
+        'Bravura Text',
         'Finale Ash',
         'Finale Broadway',
         'Finale Jazz',
         'Finale Maestro',
         'GonvilleSmufl',
         'Gootville',
+        'Gootville Text',
+        'Leipzig',
         'Leland',
+        'Leland Text',
         'MuseJazz',
         'Petaluma',
+        'Petaluma Text',
+        'Sebastian',
+        'Sebastian Text',
       ];
 
-      // The OTF files have weird names, so we need to map them to the font names.
+      // If the value is null, the folder/filename.otf are the same as the lower-cased font name.
+      // Other OTF files have custom names, so we map them to the font names.
       const textFonts = {
-        Academico: 'academico/academico.otf',
-        'Roboto Slab': 'robotoslab/robotoslab-regular-400.otf',
-        Edwin: 'edwin/edwin-roman.otf',
-        'Finale Maestro Text': 'finalemaestrotext/finalemaestrotext-regular.otf',
-        'MuseJazz Text': 'musejazztext/musejazztext.otf',
-        'Petaluma Script': 'petalumascript/petalumascript.otf',
+        Academico: null,
+        Edwin: 'edwin-roman',
+        'Finale Ash Text': null,
+        'Finale Broadway Text': null,
+        'Finale Jazz Text': null,
+        'Finale Maestro Text': 'finalemaestrotext-regular',
```

Comment:
These are the text fonts suggested by Finale to pair with each Finale font.

The naming convention does not match the Bravura Text, Leland Text, Petaluma Text, etc.... Those fonts are actually music engraving fonts designed for text heavy documents.